src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -268,21 +302,25 @@ function listenToMouseMove(event) {
         currentYear.value = futureYear
     }
 }
+
 function releaseCursor() {
     yearCursorIsGrabbed = false
     window.removeEventListener('mousemove', listenToMouseMove)
     window.removeEventListener('touchmove', listenToMouseMove)
     window.removeEventListener('mouseup', releaseCursor)
     window.removeEventListener('touchend', releaseCursor)
-    if (isPristine.value && previewYear.value !== currentYear.value) {
-        isPristine.value = false
-    }
-    store.dispatch('setPreviewYear', { year: currentYear.value, ...dispatcher })
+
+    // We call this explicitly here. So far, the store wasn't updated,
+    // as this was prevented by the yearCursorIsGrabbed switch.
+    dispatchCurrentYearToStore()
+    // another cool way would be to use a debounce on the dragging. Then
+    // the map gets updated when the user stops dragging, and not just when
+    // they release the mouse
```

Comment:
The debouncing is a very good remark and good idea ! On the old viewer if you grab the slider move it and keep it dragged then the year is changed while now on the view you have to let go to set the year. Note on currently we have a flickering issue with the slider that I'm fixing with PB-518. So it would be very nice to do the deboucing.